spec/lib/whitehall_importer/integrity_checker/change_history_check_spec.rb

```diff
@@ -0,0 +1,114 @@
+RSpec.describe WhitehallImporter::IntegrityChecker::ChangeHistoryCheck do
+  let(:published_edition) { build(:edition, document_type_id: "news_story", state: "published") }
+  let(:draft_edition) { build(:edition, document_type_id: "news_story") }
+  let(:public_timestamp) { Time.zone.now.yesterday.rfc3339 }
+
+  describe "#match?" do
+    it "returns true if the proposed payload matches" do
```

Comment:
Should this be "returns true if the proposed change history matches the Publishing API history for a non-draft edition"?